packages/commonwealth/server/routes/canvas.ts

```diff
@@ -0,0 +1,69 @@
+import { AppError } from 'common-common/src/errors';
+import type { DB } from '../models';
+import type { TypedRequestBody, TypedResponse } from '../types';
+import { success } from '../types';
+import * as Sequelize from 'sequelize';
+
+type CanvasDataReq = {
+  query: {
+    before: string
+  }
+};
+type CanvasDataResp = {};
+
+export const getCanvasData = async (
```

Comment:
When does this route get hit / for what use case?

Reply:
The p2p node polls this endpoint for the latest signed actions published from Common.